tests/features/steps/wrappers.py

```diff
@@ -4,9 +4,11 @@
 import re
 
 
-def expect_exact(context, expected, timeout):
+def expect_exact(context, expected, timeout, ignore_before=False):
     try:
         context.cli.expect_exact(expected, timeout=timeout)
+        if not ignore_before:
```

Comment:
well this is the whole point of this branch, I'd like to match as much as possible from mycli and assert if something unexpected is outputted.